sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
Also @yogsothoth  - now there are merge conflicts. I hate to bother you, but could you please take care of them?